packages/@lwc/template-compiler/src/parser/attribute.ts

```diff
@@ -37,14 +38,12 @@ import {
 
 import { isCustomElement } from '../shared/ir';
 
-function isQuotedAttribute(rawAttribute: string) {
-    const [, value] = rawAttribute.split('=');
-    return value && value.startsWith('"') && value.endsWith('"');
+function isQuotedAttribute(rawValue: string): boolean {
+    return rawValue ? rawValue.startsWith('"') && rawValue.endsWith('"') : false;
 }
 
-function isEscapedAttribute(rawAttribute: string) {
-    const [, value] = rawAttribute.split('=');
-    return !value || !(value.includes('{') && value.includes('}'));
+function hasMatchingBrackets(rawValue: string) {
+    return rawValue && rawValue.includes('{') && rawValue.includes('}');
```

Comment:
Do we care if this returns true for the string "he}ll{o"?

Reply:
@ekashida is right, we might need a more complex logic here if we want to check if curlies are matching.